packages/client/src/pages/Destination/DestinationPage.scss

```diff
@@ -0,0 +1,136 @@
+        background-color: rgba($tc-black, 0.5);
+
+        .dest-name {
+            font-family: $logo-font-family;
+            margin: 0;
+            font-size: $logo-font-size;
+            color: $tc-white;
+        }
+    }
+
+    .border {
+        position: absolute;
+        bottom: -50px;
+        width: 100%;
+        height: 100px;
+        fill: $tc-white;
+    }
+}
+
+.dest-page-content-wrapper {
+    box-sizing: border-box;
+    background-color: $tc-white;
+    padding: 0 0 10rem 0;
+    min-height: 100vh;
+    margin: auto;
+    box-shadow: -20px 1.5rem 2rem 0px rgba(0, 0, 0, 0.08);
+    font-family: $default-font-family;
+    color: $mid-gray;
+
+    .content {
+        font-family: $default-font-family;
+        font-size: $default-font-size;
+        max-width: 1280px;
+        margin: 0 auto;
+        padding: 3.125rem 1.5rem 0 1.5rem;
+    }
+
+    .dest-page-title {
+        @include title-border-bottom($powder-blue, $crayola-yellow);
+        font-size: $subtitle-font-size;
+        width: fit-content;
+    }
+
+    .dest-details {
+        display: flex;
+
+        .dest-info {
+            flex-basis: 30%;
+            display: flex;
+            flex-direction: column;
+            border: 1px solid $light-gray;
+            border-radius: 8px;
+            padding: 1rem;
+            margin-right: 0.5rem;
+            margin-bottom: auto;
+            .dest-info-item {
+                margin-bottom: 0.5rem;
+                display: flex;
+
+                svg {
+                    flex-shrink: 0;
+                }
+
+                .info-text {
+                    margin: 0 0 0 0.5rem;
+                }
+
+                .router-link {
+                    text-decoration: none;
+                    border-bottom: 2px solid $tc-white;
+                    &:hover {
+                        border-bottom-color: $powder-blue;
+                    }
+                    transition: border-bottom-color 0.3s ease-out;
+                }
+            }
+
+            .nearby-dest-item {
+                border: 1px solid $light-gray;
+                border-radius: 8px;
+                margin-bottom: 0.5rem;
+            }
+        }
+
+        .dest-map-wrapper {
+            flex-basis: 70%;
+            margin-left: 0.5rem;
+        }
+
+        .dest-ratings > * {
+            margin-bottom: 0.5rem;
+        }
+    }
+
+    .associated-trecipes-wrapper {
+        display: flex;
+        flex-direction: column;
+        .associated-trecipes-list {
+            list-style: none;
+            padding: 0;
+            display: flex;
+            overflow-x: auto;
+            width: 100%;
+
+            .associated-trecipe-item {
+                &:nth-of-type(1) {
+                    margin-left: 0;
+                }
+
+                flex-shrink: 0;
+                width: 23rem;
+                margin-left: 1rem;
+            }
+        }
+
+        ::-webkit-scrollbar {
+            width: 0;
+        }
```

Comment:
We can hide the scrollbar when it is vertical, what about when it is horizontal? I think most people won't know how scroll horizontally without the scrollbar 😆

Reply:
I see... that's a good point, I didn't know horizontal scroll was ctrl + mouse wheel till just recently hahaha. Maybe I'll bring back the scroll bar for this one.

Reply:
![image](https://user-images.githubusercontent.com/33335963/89097513-85740e80-d3ad-11ea-823d-b33ae0fdf594.png)
Ok! Looks like this now

Reply:
Thanks, I know it doesn't look great, but this is more usable 😑